### DISCUSSION TITLE
Undervolt values

### DISCUSSION THREAD

**Discussion by Inaiso** [seerge/g-helper#686] [General]
What values for undervolts are everyone able to get? Just wanted a general idea of how low of undervolt everyone is able to get to.

**Comment by seerge** [comment]
My unit stays stable somewhere between -10 and -15. And based on what I was reading on reddit it seem to be most typical case. But some people with 2023 models seem to max out in current UI to -30 :)

**Comment by NJinsei** [comment]
Flow X16 6900HS -30 UV on CPU and -17 on IGPU
